frontend/src/scenes/CallingPage.tsx

```diff
@@ -241,10 +256,15 @@ export default function CallingPage({ locale, routePath }: SceneProps) {
     setHasUserDisconnected(true);
   }, [activeConnection]);
 
-  const exitCallingPage = useCallback(() => {
-    disconnectCall();
-    userService?.setShouldSleep(true);
-    contactService?.setActiveContact(null);
+  const handleExitCallButton = useCallback(() => {
+    if (!hasUserDisconnected) {
+      disconnectCall();
+    }
+    if (activeConnection) {
+      openFeedbackDialog();
+    } else {
+      exitCallingPage();
+    }
```

Comment:
Comment on the commit message: Even if the behaviour can be inferred from the code, it is usually helpful to describe:

- the problem you are solving
- the behaviour of your code that solves the problem
- breaking changes
- possible things to watch out for
- possible contentious things in the commit

This makes it a lot easier for the reviewer because the reviewer needs to know the answers to a lot of questions, including:

- what does your code do?
- do we want the code to do this?
- is the problem solved?